skills/research/web-research-agents/scripts/web_research_agent.py

```
@@ -0,0 +1,124 @@
+    if not args.prompt:
+        parser.error("prompt is required unless --list is used")
+
+    if args.agent not in catalog:
+        raise SystemExit(f"unknown agent: {args.agent}. available: {', '.join(sorted(catalog))}")
+
+    cmd = build_command(catalog[args.agent], args.prompt)
+    if args.dry_run:
+        print(json.dumps({"command": cmd}, ensure_ascii=False, indent=2))
+        return 0
+
+    completed = subprocess.run(cmd, text=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, timeout=args.timeout)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)